src/op_handlers/near_call.rs

```diff
@@ -21,7 +21,7 @@ pub fn near_call(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmError> {
 
     let current_frame = vm.current_frame_mut()?;
 
-    current_frame.pc += 1; // The +1 used later will actually increase the pc of the new frame
+    //current_frame.pc += 1; // The +1 used later will actually increase the pc of the new frame
```

Comment:
We do +1 when calling a nearcall and then -1 when returning, so its not necessary